pkg/clientip/clientip.go

```diff
@@ -30,7 +30,10 @@ import (
 // 1. no loadbalancer, local testing
 // 2. behind Google Cloud LoadBalancer (as in cloudrun)
 //
-// At this time we have no need to complicate it further
+// Note that in particular we do not support hitting the CloudRun endpoint
+// directly (though we could easily do so).
```

Comment:
Yeah, we should not do support that, no one should know the cloud run endpoint, that would be a security issue. Cloud Armor is on the GCLB.

Reply:
Thanks - updated the note to incorporate the reasoning.